index.html

```diff
@@ -1,290 +1,428 @@
+                    </label>
+                  </li>
+                  <li class="slide-trigger">
+                    <label for="img-3" id="img-dot-3">
+                      <img src="assets/images/ex-icon-05.png" /> Send a Slack message
+                    </label>
+                  </li>
+                  <li class="slide-trigger">
+                    <label for="img-4" id="img-dot-4">
+                      <img src="assets/images/ex-icon-02.png" /> Utilize a shared workspace between jobs
+                    </label>
+                  </li>
+                </ul>
+              </div>
+
+            </div> <!-- /column -->
+          </div>
 
-  twitter.run()
-})
-              </pre>
-          </ul> <!-- /orbit -->
-
-        </div> <!-- /column -->
-
-        <div class="medium-5 hidden-for-small-only columns">
-          <h4>Example use cases:</h4>
-          <ul>
-            <li class="slide-trigger"><a data-orbit-link="headline-1" aria-selected="true"><img src="assets/images/ex-icon-01.png" /> Unit tests for a Github push</a>
-            </li>
-            <li class="slide-trigger">
-              <a data-orbit-link="headline-2"><img src="assets/images/ex-icon-07.png" /> Updating a cosmosDB database</a>
-            </li>
-            <li class="slide-trigger">
-              <a data-orbit-link="headline-3"><img src="assets/images/ex-icon-05.png" /> Sending a Slack message</a>
-            </li>
-            <li class="slide-trigger">
-              <a data-orbit-link="headline-4"><img src="assets/images/ex-icon-06.png" /> Sending a Twitter DM</a>
-            </li>
-          </ul>
+            <div class="column is-half code-examples">
+              <h4>Example use cases:</h4>
 
-          <p>Check out <a href="https://docs.brigade.sh/" target="_blank">our docs</a> for more. <img src="assets/images/arrow.png" class="arrow" /></p>
+              <p>Check out <a href="https://docs.brigade.sh/" target="_blank">our docs</a> for more. <img src="assets/images/arrow.png" class="arrow" /></p>
+            </div>          
+    
+          </div>
         </div>
       </div>
+    </section>
 
-      <div class="center">
-        <a class="button rounded center primary" href="https://github.com/brigadecore/brigade" target="_blank">
-          Get Brigade
-        </a>
+    <section id="kubernetes" class="brigade-cluster content retrobox">
+      <div class="row">
+        <div class="columns">
+          <div class="is-half column">
+    
+            <h2>Runs inside your cluster</h2>
+            <p>By running Brigade as a service inside your <a href="https://kubernetes.io/" target="_blank">Kubernetes</a> cluster, you can harness the power of millions of available Docker images.</p>
+  
+            <p>Brigade can be used to chain containers together to build processing pipelines.</p>
+            <ul class="fancy">
+              <li>Watch for incoming requests from services like GitHub, Docker, and Trello</li>
```

Comment:
```suggestion
              <li>Watch for incoming requests from services like GitHub, Docker, and BitBucket</li>
```
(per https://github.com/brigadecore/brigade-www/pull/55#issuecomment-975988830)